### ISSUE TITLE
Where is Spyder 6.0.3 app on Mac ??

### ISSUE THREAD

**Issue by wissme** [spyder-ide/spyder#23248]
Hi ! i just installed Spyder 6.0.3 on my MAC mini M1 (MacOS 15.1.1), it seems to finish ok but there is no application Spyder 6 in the /Applications folder. As I installed for myself only, I can see several spyder 6 apps in my own Applications folder but they should be the previous 6.0.2 versions (not numbered, they SHOULD be) : "Spyder 6 (anaconda3).app" from DEC 7, "Spyder 6 (cython).app" NOV 3, "Spyder 6 (simul).app" OCT 27 and "Spyder 6.app" NOV 1. ---> Please add the version number "Spyder 6.x.x.app"...
They seem to be tied to some of my Anaconda envs, right ?
So WHERE is my new Spyder 6.0.3 app ? Or do I miss something ?
Thanks,
   -w

**Comment by ccordoba12** [comment]
Hey @wissme, thanks for reporting. You said:

>  I can see several spyder 6 apps in my own Applications folder but they should be the previous 6.0.2 versions (not numbered, they SHOULD be) : "Spyder 6 (anaconda3).app" from DEC 7, "Spyder 6 (cython).app" NOV 3, "Spyder 6 (simul).app" OCT 27 and "Spyder 6.app" NOV 1

It should be `Spyder 6.app`

> Please add the version number "Spyder 6.x.x.app"...
They seem to be tied to some of my Anaconda envs, right ?

Correct, that's the case for all your conda envs.

> Please add the version number "Spyder 6.x.x.app"...

We don't think that's necessary simply because all your other apps are tied to the env name, whereas our standalone app isn't. In addition, I'm not sure if it's even technically possible.

Let us know if this clarifies the situation for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.

**Comment by ptoche** [comment]
Apparently the "Spyder 6.app" goes into the user-level Applications directory, rather than the usual Applications directory...

**Comment by ccordoba12** [comment]
Correct. If you're looking how to remove it, you need to run the command `uninstall-spyder` in a terminal.

**Comment by ptoche** [comment]
> Correct. If you're looking how to remove it, you need to run the command `uninstall-spyder` in a terminal.

thank you for your feedback Carlos. I moved it manually to the Applications directory and it appears to still work as expected.  Having said that, Spotlight Search can find it either way, so I could have left it there.
